src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/04-element-handle.md

```diff
@@ -82,7 +82,7 @@ export default async function () {
 
     await Promise.all([page.waitForNavigation(), submitButton.click()]);
     check(page, {
-      header: page.locator('h2').textContent() == 'Welcome, admin!',
+      'header': page => page.locator('h2').textContent() == 'Welcome, admin!',
```

Comment:
Asking because of curiosity: I wonder why do we need to make the `header` property to `'header'`?

Reply:
Only so that it matches what is in the [docs](https://k6.io/docs/javascript-api/k6/check/). Happy to change it to `header`, what's the "correct" way?

Reply:
I looked into it and discovered the correct way is 'header' :)

Reply:
Ah, ok 😅 I just changed it, but I'll change it back 👍

Reply:
I've resolved the comment on `page` to `p` in a105c8c8c36d205770443d992c7cf2b58d5db5a7. I have left `'header'` as `header` 👍